docs/source/en/using-diffusers/ip_adapter.md

```diff
@@ -231,10 +234,21 @@ export_to_gif(frames, "gummy_bear.gif")
 </hfoption>
 </hfoptions>
 
+## Configure parameters
+
+There are a couple IP-Adapter parameters that are useful to know about and can help you with your image generation tasks. These parameters can make your workflow more efficient or give you more control over image generation.
```

Comment:
```suggestion
There are a couple of IP-Adapter parameters that are useful to know about and can help you with your image generation tasks. These parameters can make your workflow more efficient or give you more control over image generation.
```

I know both "couple parameters" and "couple of parameters" are widely used but I guess the latter is slightly more preferable in the educational context?